sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -623,7 +623,7 @@ def list_certificate_versions(self, name: str, **kwargs: "**Any") -> AsyncIterab
         :type name: str
         :returns: An iterator like instance of CertificateBase
         :rtype:
-         typing.Generator[~azure.keyvault.certificates.models.CertificateBase]
+         Generator[~azure.keyvault.certificates.models.CertificateBase]
```

Comment:
Don't list methods actually return `ItemPaged`?

Reply:
you're right, fixing that now thanks!